bin/verilator

```diff
@@ -309,6 +310,7 @@ detailed descriptions in L</"VERILATION ARGUMENTS"> for more information.
     --getenv <var>              Get environment variable with defaults
     --help                      Display this help
      -I<dir>                    Directory to search for includes
+     -j <value>                 Parallelism for --build
```

Comment:
-j <jobs>

Reply:
thanks, fixed in 424b7e6d7ded4c5cef558fcde7505f24dab0dcf3